config.go

```diff
@@ -7,9 +7,35 @@ import (
 	"time"
 )
 
+// These are the versions of the protocol (which includes wire protocol as
+// well as Raft-specific log entries) that this server can _understand_. Use
+// the ProtocolVersion member of the Config object to control the version of
+// the protocol to use when _speaking_ to other servers.
+//
+// Version History
+//
+// 0: Unversioned original protocol spoken until Q2 2016.
+// 1: Added server IDs and a new peer change mechanism via a new LogConfiguration
+//    log entry type. All servers must be running >= 1 in order to support new
+//    staging and nonvoter modes for servers.
+const (
+	ProtocolVersionMin uint8 = 0
+	ProtocolVersionMax       = 1
```

Comment:
Is `ProtocolVersionMax` ever serialized as a specific type?  Does it make sense to just declare this as a `uint32` so that the type has a fixed width?